src/Java/Video/OggTheora.java

```diff
@@ -0,0 +1,62 @@
+import vtk.vtkNativeLibrary;
+import vtk.vtkImageCanvasSource2D;
+import vtk.vtkOggTheoraWriter;
+
+    
+public class OggTheora 
+{
+  // -----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+
+    String fileName = "test.avi";
+    for(int i = 0; i < args.length; ++i)
```

Comment:
Give the user the opportunity to input a file name. Look at WriteVTP.java and for a suggested name use `OggTheora.avi`